### PR TITLE
machine: introduce env variable for skipping checks specific only for installed images

### PR DESCRIPTION
**Pull request by KKoukiou**
This option is needed for the anaconda new UI POC tests.

### CODE REVIEW

**Comment by KKoukiou** [comment]
I guess we don't want to document this option yet in test/README in cockpit. It's use case is very limited anyway.

File: machine/machine_core/ssh_connection.py [hunk 1]
```diff
@@ -103,7 +103,11 @@ def wait_user_login(self):
         while (tries_left > 0):
             try:
                 with timeoutlib.Timeout(seconds=30):
+                    running_installer = os.getenv("TEST_RUNNING_INSTALLER", False)
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply machine/machine_core/ssh_connection.py:106]: Yes /run/nologin is always present.

```
anaconda root@fedora ~]# systemctl status systemd-user-sessions.service
○ systemd-user-sessions.service - Permit User Sessions
     Loaded: loaded (/usr/lib/systemd/system/systemd-user-sessions.service; static)
     Active: inactive (dead)
       Docs: man:systemd-user-sessions.service(8)
```

[the remark this answers is not in the document]
Comment (KKoukiou) [reply machine/machine_core/ssh_connection.py]: updated!

**Review by martinpitt — APPROVED** [review]
Thanks! I trigger two integration tests, just to be on the safe side.

**Pull request merged**